packages/annotations/src/search.ts

```diff
@@ -105,3 +108,160 @@ export function searchAnnotations(
     });
   });
 }
+
+export interface IVoteResourceObject {
+  id: number;
+  type: "votes";
+  attributes: {
+    upVotes: number;
+    downVotes: number;
+  };
+}
+
+/**
+ * ```js
+ * import { searchAnnotationVotes } from "@esri/hub-annotations";
+ * //
+ * searchAnnotationVotes({ url: annotationsUrl + "/0"},
+ *                         annotation: {
+ *                            id: "Annotation1",
+ *                            type: "annotations",
+ *                            attributes: {description: "Great place!", ...}
+ *                          })
+ *   .then(response => {
+ *     //   data: [{
+ *     //     id,
+ *     //     type: "votes",
+ *     //     attributes: {
+ *     //       upVotes: 3,
+ *     //       downVotes: 0
+ *     //     }
+ *     //   }]
+ *    });
+ * ```
+ * Query for up and down votes on a comment from ArcGIS Hub.
+ * @param requestOptions - request options that may include authentication
+ * @param annotation - the annotation for which votes need to be counted
+ * @returns A Promise that will resolve with summary statistics for the specified annotation from the annotation service for a Hub enabled ArcGIS Online organization.
+ */
+export function searchAnnotationVotes(
+  requestOptions: IQueryFeaturesRequestOptions,
+  annotation: IAnnoFeature
+): Promise<{ data: IVoteResourceObject[] }> {
```

Comment:
lets rename this function and refactor it so that only one input object needs to be passed.
```ts
export function searchSingleAnnotationVotes(
  requestOptions: ISearchAnnoRequestOptions // IRequestOptions + url + IAnnoFeature
): Promise<{ data: IVoteResourceObject[] }> {
```
we can still cast an `IQueryFeaturesRequestOptions` internally, but i don't _think_ we want/need to allow folks to pass other query filters.